getting-started/first-steps-sl.md

```diff
@@ -0,0 +1,93 @@
+---
+title: First Steps
+page_title: First Steps
+description: First Steps
+slug: getting-started-first-steps-sl
+tags: get,started,first,steps
+published: True
+position: 0
+---
+
+# First Steps
+
+This article explains how to get the Telerik UI for {{ site.framework_name }} controls in your project and start using them quickly.
+
+Once you have your first simple control up and running, take a look at the [next steps](#next-steps) section to start exploring the control functionality in more detail.
+
+## Download the Controls
+
+The easiest way to get the controls to your development machine is to use the [Progress Control Panel](https://www.telerik.com/download-trial-file/v2/control-panel) or to download the [automated MSI installer]({%slug installation-installing-from-msi%}) from [your telerik.com account](https://www.telerik.com/account/product-download?product=RCSL).
+
+>caption Figure 1: Download automated (.msi) installer
+
+![Download automated installer Telerik_UI_for_Silverlight_<version>_Dev.msi](images/download-msi-sl.png "Telerik_UI_for_Silverlight_<version>_Dev.msi")
+
+> If you are not a customer, you can [download a free, fully functional trial](https://www.telerik.com/download-trial-file/v2-b/ui-for-silverlight) and the same options will apply to you as well.
+
+>tip The following article can help you choose the installation type that is most suitable for your needs and preferences: [Which File Do I Need to Install]({%slug installation-installing-which-file-do-i-need%}).
+
+## Create a UI for {{ site.framework_name }} Project
```

Comment:
This article seems like it was created only for Silverlight. In this case you don't need the 'framewor_kname' tag here.